livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -284,6 +288,9 @@ def __init__(
             ValueError: If the API key is not provided and cannot be found in environment variables.
         """
         super().__init__()
+        self._capabilities = Capabilities(
+            supports_chat_ctx_manipulation=False,
```

Comment:
re (from Slack): this isn't used on OpenAI

Reply:
oh wait, It should be True for Openai, because we need to few operations such as truncate, update conversations items etc.